client.go

```diff
@@ -159,12 +187,33 @@ func (c *Client[Req, Res]) CallServerStream(ctx context.Context, request *Reques
 	if c.err != nil {
 		return nil, c.err
 	}
+	callInfo, callInfoOk := getClientCallInfoFromContext(ctx)
```

Comment:
I think this can be shifter to `newConn` to support client and bidi as well.

Reply:
Maybe that should be added in https://github.com/smaye81/connect-go/pull/1, which implements client and bidi?